tests/test_utils.py

```diff
@@ -62,3 +62,23 @@ def test_get_workspace_uri_request_exception(mock_get):
     TestCase.assertRaises(
         TestCase, requests.exceptions.RequestException, get_workspace_uri
     )
+
+
+@mock.patch("builtins.open")
+@mock.patch("json.load", mock.MagicMock(side_effect=[{"test": "test"}]))
```

Comment:
`MagicMock` is automatically used by `patch`, just add directly the side effect:

```python
@mock.patch("json.load", side_effect=[{"test": "test"}])
```